index.html

```diff
@@ -136,6 +136,204 @@ <h2>
+          </li>
+        </ul>
+      </p>
+      <p>
+        A [=gamepad=] has a <dfn>gamepad identifier string</dfn>, a
+        human-readable string that identifies the brand or style of
+        [=gamepad=]. The content is decided by the [=user agent=].
+      </p>
+      <section>
+        <h3>
+          Input control layout
+        </h3>
+      <p>
+        A [=gamepad=] may have an <dfn>input control layout</dfn> that describes the position, orientation and type of each [=input control=] on the [=gamepad=].
+        The [=user agent=] is responsible for recognizing when a [=gamepad=] <dfn data-lt="corresponds with">corresponds with a standard layout</dfn>, meaning the [=gamepad=] has an [=input control layout=] that enables it to be used interchangeably with other [=gamepads=] that correspond with the same standard layout.
+        The [=user agent=] SHOULD consider a layout to correspond with a standard layout if its [=input controls=] have approximately the same relative positions and orientations as [=input controls=] described in the standard layout.
+      </p>
+      <p>
+        The [=user agent=] typically cannot directly inspect the [=input control layout=] for a [=gamepad=] and MAY use heuristics to decide the layout.
+        The [=user agent=] SHOULD consider the device identifiers when deciding whether a [=gamepad=] [=corresponds with a standard layout=].
+        If the system assigns a label to each [=input control=] and the labels imply a particular layout then the [=user agent=] SHOULD consider the [=gamepad=] to have that layout.
+        When there is a standard model and an accessible model with the same [=input controls=], the [=user agent=] SHOULD consider the accessible model to have the same [=input control layout=] as the standard model.
+      </p>
+      </section>
+      <section>
+        <h3>
+          Input values
+        </h3>
+        <p>
+          Each input control has one or more associated [=input values=], which are numerical values that represent the current state of the control.
+          [=Input values=] can update at any time.
+          The [=user agent=] is responsible for detecting when [=input values=] have updated and SHOULD try to minimize the delay between the update and when the updated values are read.
+        </p>
+        <p>
+          Reading an [=input value=] returns its <dfn>logical value</dfn>, an unscaled numerical representation of the current state.
+          An [=input value=] also has a <dfn>logical minimum</dfn> and <dfn>logical maximum</dfn> which define the minimum and maximum in-bounds [=logical values=].
+        </p>
+        <p>
+          An [=input value=] may have an associated <dfn>HID usage identifier</dfn>, a 32-bit value that identifies the type of data represented by the input value.
+          A HID usage does not precisely describe the [=input control layout=], but by convention many [=gamepads=] with similar layouts use similar usages.
```

Comment:
> by convention many [=gamepads=] with similar layouts use similar usages

It would be great to a resource with the list of conventional HID usage identifiers. Even if it is not strictly standardizes, it would be helpful to have such a list for reference.

Reply:
For a complete list of usage identifiers, there's https://www.usb.org/document-library/hid-usage-tables-16 (a 443-page PDF)?